tests/perf-test/ReportScreen.perf-test.js

```diff
@@ -158,6 +192,10 @@ test('[ReportScreen] should render ReportScreen with composer interactions', ()
     const reportActions = ReportTestUtils.getMockedReportActionsMap(1000);
     const mockRoute = {params: {reportID: '1'}};
 
+    const navigation = {
+        addListener,
+    };
```

Comment:
Since it has only one element, can we just do:

```suggestion
    const navigation = {addListener};
```